chapters/modules/chapter_2/pages/contracts/src/vote.cairo

```diff
@@ -0,0 +1,191 @@
+/// Vote contract allows users to submit their votes (1 for Yes/0 for No) on a proposal.
```

Comment:
@l-henri @gyan0890 this is a simple voting smart contract to explain Cairo components.